src/main/java/org/example/springproject/dto/user/UserRegistrationRequestDto.java

```diff
@@ -0,0 +1,23 @@
+package org.example.springproject.dto.user;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import org.example.springproject.validation.FieldMatch;
+
+@Data
```

Comment:
please use concreate annotations (getter, setter etc) instead of this compound one. Fix the same issue in all places